pages/platform/kubernetes-k8s/backing-up-cluster-with-velero/guide.en-asia.md

```diff
@@ -505,14 +483,12 @@ NAME         STATUS   VOLUME                                     CAPACITY   ACCE
 nginx-logs   Bound    pvc-71528d91-fd03-46c4-99d6-557d3e8fe9da   1Gi        RWO            cinder-classic   35s
 
 $ kubectl get pv -n nginx-example
-NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM
+NAME                                      CAPACITY  ACCESS MODES  RECLAIM POLICY STATUS CLAIM
   STORAGECLASS     REASON   AGE
-pvc-71528d91-fd03-46c4-99d6-557d3e8fe9da   1Gi        RWO            Delete           Bound    nginx-example/nginx-logs   cinder-classic            39s
+pvc-71528d91-fd03-46c4-99d6-557d3e8fe9da  1Gi       RWO           Delete         Bound  nginx-example/nginx-logs cinder-classic  39s
 </code></pre>
 
+## Where do we go from here
```

Comment:
Why removing the '?' ?

Reply:
Because my IDE is configured to use a Markdown linter and the rule [MD026](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md026---trailing-punctuation-in-heading) do not accept trailing punctuation to finish heading tag.
If you prefer, I can cancel the change.